run.py

```diff
@@ -76,10 +76,30 @@
 
             if model.calculate_importance:
                 probas, importances = model.classify(bug, probabilities=True, importances=True)
-
                 feature_names = model.get_feature_names()
-                for i, (importance, index, is_positive) in enumerate(importances):
-                    print(f'{i + 1}. \'{feature_names[int(index)]}\' ({"+" if (is_positive) else "-"}{importance})')
+
+                if len(importances.shape) < 2:
+                    for i, (importance, index, is_positive) in enumerate(importances):
+                        print(f'{i + 1}. \'{feature_names[int(index)]}\' ({"+" if (is_positive) else "-"}{importance})')
+                else:
+                    # Number of classes to be printed
```

Comment:
Maybe it would be best here to only print the features which contributed to the selection of the predicted class.

Reply:
What an user wants to know here is "why did the classifier select this class?"